config/notifiers.go

```diff
@@ -579,3 +589,29 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return nil
 }
+
+// NewRelicConfig configures notifications via NewRelic.
+type NewRelicConfig struct {
+	NotifierConfig `yaml:",inline" json:",inline"`
+	HTTPConfig     *commoncfg.HTTPClientConfig `yaml:"http_config,omitempty" json:"http_config,omitempty"`
+	APIKey         Secret                      `yaml:"api_key,omitempty" json:"api_key,omitempty"`
+	APIURL         *URL                        `yaml:"api_url,omitempty" json:"api_url,omitempty"`
+	Message        string                      `yaml:"message,omitempty" json:"message,omitempty"`
+	Description    string                      `yaml:"description,omitempty" json:"description,omitempty"`
+	Source         string                      `yaml:"source,omitempty" json:"source,omitempty"`
+	Details        map[string]string           `yaml:"details,omitempty" json:"details,omitempty"`
+	Headers        map[string]string           `yaml:"headers,omitempty" json:"headers,omitempty"`
```

Comment:
Why do we need this?

Reply:
@roidelapluie  New Relic endpoint expects API key in the header.  It accepts one of the two headers currently: `Api-Key` and ` X-License-Key `.    Having **Headers** allows user to configure the one they prefer. It also provides provide some flexibilities in case the header requirement changes in the future.  Thanks.

Reply:
I removed the Headers setting option. User will use api_key to provide license key explicitly.